cpp/daal/include/services/internal/sycl/types.h

```diff
@@ -67,11 +67,11 @@ inline String getKeyFPType()
 {
     if (IsSameType<algorithmFPType, float>::value)
     {
-        return String(" -D algorithmFPType=float -D algorithmFPType2=float2 -D algorithmFPType4=float4 ");
+        return String(" -D algorithmFPType=float -D algorithmFPType2=float2 -D algorithmFPType4=float4 -D MAX_FPTYPE=FLT_MAX");
```

Comment:
What's the purpose of this addition? 
The name `MAX_FPTYPE` is confusing.  What value should it contain?